### DISCUSSION TITLE
how do lsun-bedroon need the data directory？

### DISCUSSION THREAD

**Discussion by zhjw0927** [open-mmlab/mmgeneration#457] [Datasets] [answered]
I have downloaded the lsun-bedroom dataset on the official website. Its specific directory structure is as follows. 
![Screenshot from 2022-07-04 05-56-36](https://user-images.githubusercontent.com/42128459/177131203-9867d269-6085-41fd-becf-945dc54f4b4f.png)
![Screenshot from 2022-07-04 05-56-42](https://user-images.githubusercontent.com/42128459/177131211-e61f8c58-4dd3-4cdf-a712-3b1eb674222c.png)

Its suffix is mdb, which is not in the `_VALID_IMG_SUFFIX = ('.jpg', '.png', '.jpeg', '.JPEG')`.
Can you show me the directory structure of lsun-bedroom?

**Comment by LeoXing1996** [comment, marked as answer]
You can use this [script](https://github.com/NVlabs/stylegan3/blob/main/dataset_tool.py) to generate image files. 
We will add this script to our repo later as well.
